frontend/packages/console-app/src/components/guided-tours/GuidedTourDrawer.scss

```diff
@@ -0,0 +1,12 @@
+.co-guided-tour-drawer {
+  &__panel-content {
+    overflow-y: auto !important;
+    z-index: auto;
```

Comment:
Not sure why either of these are needed right now.

Reply:
removed class for drawer